vignettes/wpgsd_corr_example.Rmd

```diff
@@ -66,197 +70,90 @@ event_tbl %>%
+### Example 2 - Correlation of different analyses within the same population
+Let's consider another simple situation, we want to compare single population, for example, the population 1, but in different analyses, interim and final analyses. Then  $i=1$, and to compare IA and FA, the $k$ will be $k=1$ and $k=2$. 
 The correlation matrix will be
 $$Corr(Z_{11},Z_{12})=\frac{n_{1 \wedge 1,1 \wedge 2}}{\sqrt{n_{11}*n_{12}}}$$
 The number of events are listed as
 ```{r}
 event_tb2 <- tribble(
   ~Population, ~"Number of Event in IA", ~"Number of Event in FA",
-  "Population 1", 100, 200
+  "Population 1", 100,200
 )
 event_tb2 %>%
   gt() %>%
   tab_header(title = "Number of events at each analyses in example 2")
 ```
 The the corrleation could be simply calculated as 
 $$Corr(Z_{11},Z_{12})=\frac{100}{\sqrt{100*200}}=0.71$$
+The 100 in the numerator is the overlap number of events of interim analysis and final analysis in population 1.
 ```{r}
-Corr1 <- 100 / sqrt(100 * 200)
-round(Corr1, 2)
+Corr1=100/sqrt(100*200)
+round(Corr1,2)
 ```
-### Example 3 - Cross Population Cross Analyses
+### Example 3 - Correlation of different analyses and different population
 Let's consider the situation that we want to compare population 1 in interim analyses and population 2 in final analyses. Then for different population, $i=1$ and $i=2$, and to compare IA and FA, the $k$ will be $k=1$ and $k=2$. 
 The correlation matrix will be
 $$Corr(Z_{11},Z_{22})=\frac{n_{1 \wedge 1,2 \wedge 2}}{\sqrt{n_{11}*n_{22}}}$$
 The number of events are listed as
 ```{r}
 event_tb3 <- tribble(
   ~Population, ~"Number of Event in IA", ~"Number of Event in FA",
-  "Population 1", 100, 200,
+  "Population 1", 100,200,
   "Population 2", 110, 220,
-  "Overlap in population 1 and 2", 80, 160
+  "Overlap in population 1 and 2", 80,160
+
 )
 event_tb3 %>%
   gt() %>%
   tab_header(title = "Number of events at each population & analyses in example 3")
 ```
-The the corrleation could be simply calculated as 
+The correlation could be simply calculated as 
 $$Corr(Z_{11},Z_{22})=\frac{80}{\sqrt{100*220}}=0.54$$
+The 80 in the numerator is the overlap number of events of population 1 in interim analysis and population 2 in final analysis.
 ```{r}
-Corr1 <- 80 / sqrt(100 * 220)
-round(Corr1, 2)
+Corr1=80/sqrt(100*220)
+round(Corr1,2)
 ```
-Now we know how to calculate the correlation values under different situations, and the generate_corr function was built based on this logic. We can directly calculate the results for each cross situation via the function. 
-
-First, we need a event table including the information of the cohort.
+Now we know how to calculate the correlation values under different situations, and the generate_corr() function was built based on this logic. We can directly calculate the results for each cross situation via the function. 
```

Comment:
When you cite a R function name please use single quotation, i.e., use "`generate_corr`" install of "generate_corr". Please check the entire of the Rmd file.